lib/features/email/presentation/widgets/attachment_list/attachment_list_dialog_body_builder.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:core/presentation/resources/image_paths.dart';
+import 'package:core/presentation/views/button/icon_button_web.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:model/email/attachment.dart';
+import 'package:tmail_ui_user/features/email/presentation/styles/attachment/attachment_list_styles.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/attachment_list/attachment_list_action_button_builder.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/attachment_list/attachment_list_dialog_builder.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/attachment_list/attachment_list_item_widget.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+
+class AttachmentListDialogBodyBuilder extends StatelessWidget {
+  final BuildContext context;
+  final ImagePaths imagePaths;
+  final List<Attachment> attachments;
+  final double? widthDialog;
+  final double? heightDialog;
+  final OnDownloadAllButtonAction? onDownloadAllButtonAction;
+  final OnDownloadAttachmentFileAction? onDownloadAttachmentFileAction;
+  final OnCancelButtonAction? onCancelButtonAction;
+  final OnCloseButtonAction? onCloseButtonAction;
+
+  const AttachmentListDialogBodyBuilder({
+    super.key,
+    required this.context,
+    required this.imagePaths,
+    required this.attachments,
+    this.widthDialog,
+    this.heightDialog,
+    this.onDownloadAllButtonAction,
+    this.onDownloadAttachmentFileAction,
+    this.onCancelButtonAction,
+    this.onCloseButtonAction
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    ScrollController scrollController = ScrollController();

```

Comment:
It should not be initialized in the `build` function. It can be created multiple times for each tree widget to be rebuilt